apps-common/query-conf/date-from.js

```diff
@@ -13,11 +13,10 @@ module.exports = {
 		}
 		if (!value) {
 			if (dateTo) {
-				return new db.Date(dateTo.getUTCFullYear(), dateTo.getUTCMonth(),
-						dateTo.getUTCDate() - 6);
+				return new db.Date(dateTo.getUTCFullYear());
 			}
-			// last week by default
-			return new db.Date(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate() - 6);
+			// current year by default
+			return new db.Date(now.getUTCFullYear());
```

Comment:
This will set the date close to timestamp 0 (1/1/1970). Consider the following:

```javascript
> var d = new db.Date()
> console.log(new db.Date(d.getUTCFullYear()));
1/1/1970
```

This is because in single argument mode db.Date treats it as timestamp.